experimental/devices/pca9685/pca9685.go

```diff
@@ -145,3 +145,27 @@ func (d *Dev) SetAllPwm(on, off gpio.Duty) error {
 func (d *Dev) SetPwm(channel int, on, off gpio.Duty) error {
 	return d.setPWM(led0OnL+byte(4*channel), on, off)
 }
+
+// SetFullOff sets PWM duty to 0%.
+//
+// This function uses the dedicated bit to reduce bus traffic.
+func (d *Dev) SetFullOff(channel int) error {
+	_, err := d.dev.Write([]byte{
+		led0OnL + byte(4*channel) + 3, // LEDX_OFF_H
+		0x10,                          // bit 4 is full-off
+	})
+	return err
+}
+
+// SetFullOn sets PWM duty to 100%.
+//
+// This function uses the dedicated FULL_ON bit.
+func (d *Dev) SetFullOn(channel int) error {
+	_, err := d.dev.Write([]byte{
```

Comment:
Should it check for channel to be valid?

Reply:
good point, yes it should

Reply:
Ok I'll wait for this to be added then I will integrate in.

Reply:
Thanks! I fixed it.